api/envoy/extensions/access_loggers/stream/v3/stream.proto

```diff
@@ -1,23 +1,34 @@
 syntax = "proto3";
 
-package envoy.extensions.access_loggers.stdoutput.v3;
+package envoy.extensions.access_loggers.stream.v3;
 
 import "envoy/config/core/v3/substitution_format_string.proto";
 
 import "udpa/annotations/status.proto";
 import "validate/validate.proto";
 
-option java_package = "io.envoyproxy.envoy.extensions.access_loggers.stdoutput.v3";
-option java_outer_classname = "StdoutputProto";
+option java_package = "io.envoyproxy.envoy.extensions.access_loggers.stream.v3";
+option java_outer_classname = "StreamProto";
 option java_multiple_files = true;
 option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
-// [#protodoc-title: Standard Output access log]
-// [#extension: envoy.access_loggers.stdoutput]
+// [#protodoc-title: Standard Streams Access loggers]
+// [#extension: envoy.access_loggers.stream]
 
 // Custom configuration for an :ref:`AccessLog <envoy_api_msg_config.accesslog.v3.AccessLog>`
 // that writes log entries directly to the operating system's standard output.
-message StdoutputAccessLog {
+message StdoutAccessLog {
+  oneof access_log_format {
+    // Configuration to form access log data and format.
+    // If not specified, use :ref:`default format <config_access_log_default_format>`.
+    config.core.v3.SubstitutionFormatString log_format = 1
+        [(validate.rules).message = {required: true}];
+  }
+}
+
+// Custom configuration for an :ref:`AccessLog <envoy_api_msg_config.accesslog.v3.AccessLog>`
+// that writes log entries directly to the operating system's standard error.
+message StdErrAccessLog {
```

Comment:
super nit: inconsistent case between "out" and "Err". Can we be consistent there?

Reply:
good catch! fixed in fe3bfb8cf431619e7d0dafbf32cb627785c5c740